apps/web/src/lib/contentful/index.ts

```diff
@@ -0,0 +1,11 @@
+import { createClient as createContentClient } from 'contentful'
+
+const contentfulEnvironment = process.env.CONTENTFUL_ENVIRONMENT || 'master'
```

Comment:
might be good know this is working to get the `assert` back in here with the de-strucuring and the ability to setup a preview if required.  Preview should be set to true in lower env's to allow the client to test without the need to publish